setup-heroku.sh

```diff
@@ -34,11 +34,20 @@ heroku fork --to $ENV_NAME-dev --from $ENV_NAME-prod
 heroku config:set SECRET_KEY=`python -c 'import random; print("".join([random.choice("abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)") for i in range(50)]))'` --app $ENV_NAME-dev
 heroku config:set SECRET_KEY=`python -c 'import random; print("".join([random.choice("abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)") for i in range(50)]))'` --app $ENV_NAME-staging
 
-heroku config:set DJANGO_SETTINGS_MODULE=$ENV_NAME.settings.heroku-dev --app $ENV_NAME-dev
-heroku config:set DJANGO_SETTINGS_MODULE=$ENV_NAME.settings.heroku-staging --app $ENV_NAME-staging
-
+heroku config:set AWS_SECRET_ACCESS_KEY="" --app $ENV_NAME-dev
```

Comment:
i wanted to loop through (dev staging heroku) but it turns out that arrays are a bashism...